lib/response/get/checkCacheAndReport.js

```diff
@@ -71,11 +70,7 @@ module.exports = function checkCacheAndReport(model, requestedPaths, observer,
     // 2.  The request if finished and the json key off
     // the valueNode has a value.
     if (progressive || ((progressive && results.hasValues || !progressive) && completed && valueNode !== undefined)) {
-        try {
-            observer.onNext(valueNode);
-        } catch (e) {
-            throw e;
-        }
+        observer.onNext(valueNode);
```

Comment:
The `try catch` was vacuous, right?

Reply:
It was. Originally, there was a `finally` as well, but it had been removed while leaving the try/catch in place.